src/growableHeap.js

```diff
@@ -42,6 +42,18 @@ function GROWABLE_HEAP_U32() {
   }
   return HEAPU32;
 }
+function GROWABLE_HEAP_I64() {
+  if (wasmMemory.buffer != HEAP8.buffer) {
+    updateMemoryViews();
+  }
+  return HEAP64;
+}
+function GROWABLE_HEAP_U64() {
+  if (wasmMemory.buffer != HEAP8.buffer) {
+    updateMemoryViews();
+  }
+  return HEAPU64;
+}
```

Comment:
I think this will fail on closure errors in a non-wasm64 build, since HEAP64/U64 are only defined in wasm64 builds. So this should be ifdefed on that. Then the reference to this file in `tools/building.py` will need to run the preprocessor on it to handle that ifdef, which I think can be done with `shared.read_and_preprocess`.